tests/registry_disk_test.go

```diff
@@ -83,13 +83,13 @@ var _ = Describe("RegistryDisk", func() {
 			for i := 0; i < num; i++ {
 				obj, err := virtClient.RestClient().Post().Resource("virtualmachines").Namespace(tests.NamespaceTestDefault).Body(vm).Do().Get()
 				Expect(err).To(BeNil())
-				tests.WaitForSuccessfulVMStartWithTimeout(obj, 60)
+				tests.WaitForSuccessfulVMStartWithTimeout(obj, 120)
 				_, err = virtClient.RestClient().Delete().Resource("virtualmachines").Namespace(vm.GetObjectMeta().GetNamespace()).Name(vm.GetObjectMeta().GetName()).Do().Get()
 				Expect(err).To(BeNil())
 				tests.NewObjectEventWatcher(obj).SinceWatchedObjectResourceVersion().WaitFor(tests.NormalEvent, v1.Deleted)
 			}
 			close(done)
-		}, 180)
+		}, 200)
```

Comment:
3 minutes wasn't always enough, but 3 minutes and 20 seconds is? I feel like we'll be re-visiting this at some point... :)

Reply:
possibly. I never saw the 180 second timeout hit. I did see it approach 160 seconds though which is why I increased it. This test is subject to inconsistent timing in the k8s runtime related to pod deletion/garbage-collection.

I don't think there is value in running 3 iterations of this test. I'm going to reduce it to 2 so we can lower the timeout values.